### PR TITLE
translations: Research translations for Latin American Spanish

### PR DESCRIPTION
**Pull request by Juanvulcano**

### CODE REVIEW

**Comment by smarx** [comment]
*Automated message from Dropbox CLA bot*

@Juanvulcano, it looks like you've already signed the Dropbox CLA. Thanks!

**Comment by umairwaheed** [comment]
@Juanvulcano, can you remove the unrelated commit and files?

**Comment by umairwaheed** [comment]
I think the folder name should be `LatinAmericanSpanish'.

File: translations/LatamSpanish/docs.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+For example: Latin American varieties of Spanish do not use vosotros (you, plural, informal), it is preferred the formal ustedes.
+
+The vast majority of Spanish words are universal, but some are not.
+Some examples such as: teléfono móvil / celular and ordenador / computadora,
+with the second of each pair being the Latin American form.
+
+
+Below are some words commonly used in applications:
+
+Home : Inicio
```
Comment (umairwaheed) [inline translations/LatamSpanish/docs.md]: These lines are joined in the rendered view:
screen shot 2017-01-16 at 10 17 12 am

See [here](https://github.com/Juanvulcano/zulip-gci/blob/492e29bcc6b382209bcc39667af223169dc60230/translations/LatamSpanish/docs.md).

**Comment by umairwaheed** [comment]
You probably want to name your file 'notes.md'.

**Comment by YagoGG** [comment]
Hey, @Juanvulcano!

Since the GCI is already over, we're closing this.

Thank you for your work, we appreciate it :)

**Pull request closed without merging**